app/ai_backend/test_genesis_core.py

```
@@ -0,0 +1,438 @@
+
+import pytest
+import json
+import tempfile
+import os
+from unittest.mock import Mock, patch, MagicMock
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)